static/js/views/installed-addon.js

```diff
@@ -79,8 +79,9 @@ class InstalledAddon {
           <span class="addon-settings-author">
             ${fluent.getMessage('by')} <a href="${this.homepageUrl}" target="_blank" rel="noopener">${Utils.escapeHtml(this.author)}</a>
           </span>
-          <span class="addon-settings-license">
-            (<a href="${this.licenseUrl}" target="_blank" rel="noopener">license</a>)
+          <span id="addon-license-${Utils.escapeHtmlForIdClass(this.id)}" class="addon-settings-license" 
```

Comment:
`(license)` should be moved into the translation file. It can then be used by adding `data-l10n-id="<id>"` to the element.

Reply:
I thought using `${fluent.getMessage('by')}` would be cleaner, since the translation then doesn't need braces.